<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<HTML>
 <HEAD>
   <TITLE> [Twisted-Python] If the errbacks of a canceled Deferred are called with error other than CancelledError, is this acceptable?
   </TITLE>
   <LINK REL="Index" HREF="index.html" >
   <LINK REL="made" HREF="mailto:twisted-python%40twistedmatrix.com?Subject=Re%3A%20%5BTwisted-Python%5D%20If%20the%20errbacks%20of%20a%20canceled%20Deferred%20are%0A%20called%20with%20error%20other%20than%20CancelledError%2C%20is%20this%20acceptable%3F&In-Reply-To=%3C51C0CD89.6010101%40itamarst.org%3E">
   <META NAME="robots" CONTENT="index,nofollow">
   <style type="text/css">
       pre {
           white-space: pre-wrap;       /* css-2.1, curent FF, Opera, Safari */
           }
   </style>
   <META http-equiv="Content-Type" content="text/html; charset=utf-8">
   <LINK REL="Previous"  HREF="059574.html">
   <LINK REL="Next"  HREF="059573.html">
 </HEAD>
 <BODY BGCOLOR="#ffffff">
   <H1>[Twisted-Python] If the errbacks of a canceled Deferred are called with error other than CancelledError, is this acceptable?</H1>
    <B>Itamar Turner-Trauring</B> 
    <A HREF="mailto:twisted-python%40twistedmatrix.com?Subject=Re%3A%20%5BTwisted-Python%5D%20If%20the%20errbacks%20of%20a%20canceled%20Deferred%20are%0A%20called%20with%20error%20other%20than%20CancelledError%2C%20is%20this%20acceptable%3F&In-Reply-To=%3C51C0CD89.6010101%40itamarst.org%3E"
       TITLE="[Twisted-Python] If the errbacks of a canceled Deferred are called with error other than CancelledError, is this acceptable?">itamar at itamarst.org
       </A><BR>
    <I>Tue Jun 18 15:13:45 MDT 2013</I>
    <P><UL>
        <LI>Previous message (by thread): <A HREF="059574.html">[Twisted-Python] If the errbacks of a canceled Deferred are called with error other than CancelledError, is this acceptable?
</A></li>
        <LI>Next message (by thread): <A HREF="059573.html">[Twisted-Python] If the errbacks of a canceled Deferred are	called with error other than CancelledError, is this acceptable?
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#59572">[ date ]</a>
              <a href="thread.html#59572">[ thread ]</a>
              <a href="subject.html#59572">[ subject ]</a>
              <a href="author.html#59572">[ author ]</a>
         </LI>
       </UL>
    <HR>  
<!--beginarticle-->
<PRE>On 06/18/2013 02:22 PM, Glyph wrote:
&gt;<i> Making an API that previously documented raising (or failing) 
</I>&gt;<i> exception types A, B, and C raise (or fail with) D is not necessarily 
</I>&gt;<i> a compatible change.  Making it raise (or fail with) A' (a subclass of 
</I>&gt;<i> A) is, though.
</I>&gt;<i>
</I>The API for pop3client does *not* document the expected exceptions, so 
backwards compatibility isn't really an issue here.


</PRE>

<!--endarticle-->
    <HR>
    <P><UL>
        <!--threads-->
	<LI>Previous message (by thread): <A HREF="059574.html">[Twisted-Python] If the errbacks of a canceled Deferred are called with error other than CancelledError, is this acceptable?
</A></li>
	<LI>Next message (by thread): <A HREF="059573.html">[Twisted-Python] If the errbacks of a canceled Deferred are	called with error other than CancelledError, is this acceptable?
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#59572">[ date ]</a>
              <a href="thread.html#59572">[ thread ]</a>
              <a href="subject.html#59572">[ subject ]</a>
              <a href="author.html#59572">[ author ]</a>
         </LI>
       </UL>

<hr>
<a href="https://twistedmatrix.com/cgi-bin/mailman/listinfo/twisted-python">More information about the Twisted-Python
mailing list</a><br>
</body></html>
